test/integration/testUtils.ts

```diff
@@ -68,17 +69,20 @@ export async function fetchStubbedResponse(stubbedResponse: any, fetchAction: ()
 /*
  * Do the work to start a login and get the temp cookie
  */
-async function startLogin(): Promise<[string, string]> {
-
-    const response = await fetch(
-        `${oauthAgentBaseUrl}/login/start`,
-        {
-            method: 'POST',
-            headers: {
-                origin: config.trustedWebOrigins[0],
-            },
+export async function startLogin(clientOptions: ClientOptions | null = null): Promise<[string, string]> {
+
+    const requestOptions = {
+        method: 'POST',
+        headers: {
+            origin: config.trustedWebOrigins[0],
         },
-    )
+    } as RequestInit
+
+    if (clientOptions) {
+        requestOptions.body = JSON.stringify(clientOptions)
```

Comment:
Shouldn't the parameter be named `body` instead of `clientOptions`?

Reply:
I used requestBody and responseBody for clarity